src/modules/getUserNumber.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+'use strict';
+
+const readline = require('readline');
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+function isValidUserNumber(str) {
+  if (isNaN(str)) {
+    console.log('You need to use only the digits from 0 to 9');
+
+    return;
+  }
+
+  if (str.length !== 4) {
+    console.log('You need to enter 4 digits');
+
+    return;
+  }
+
+  for (let i = 0; i < str.length; i++) {
+    if (str.slice(i + 1).includes(str[i])) {
+      console.log('Digits must be different');
+
+      return;
+    }
+  }
```

Comment:
you can return false, not just silent return

Reply:
ok thanks!